packages/kit/src/core/dev/index.js

```diff
@@ -104,7 +114,11 @@ class Watcher extends EventEmitter {
 			publicDir: this.config.kit.files.assets,
 			server: {
 				...user_config.server,
-				middlewareMode: true
+				middlewareMode: true,
+				hmr: {
+					...user_config.server?.hmr,
```

Comment:
The CI is failing and I think it's because of `?.`

Reply:
Yes, you're right.  Optional chaining was implemented in Node 14, so the Node 12 test failed.  I learn something new every day.

I haven't generated the changeset yet -- I'm a little uncertain about the options.  I think I'd only select the 'kit' package, and a 'patch' version, but I'd appreciate your confirmation.

Reply:
Yes, that's correct